src/reset.css

```
@@ -0,0 +1,7 @@
+body,
+h3,
+p {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
```

Font-family в reset.css не задается, но можно поставить стандартное значение
```suggestion
  font-family: Arial, sans-serif;
```

---

Можно задать для класса блока, который будет использовать `<html>`